test/Query.js

```diff
@@ -1166,13 +1166,13 @@ describe("Query", () => {
 		it("Should send correct request on query.exec", async () => {
 			queryPromiseResolver = () => ({"Items": []});
 			await Model.query("name").eq("Charlie").attributes(["id"]).exec();
-			expect(queryParams.ProjectionExpression).to.eql("id");
+			expect(queryParams.ProjectionExpression).to.eql("#a1");
```

Comment:
Let's also add another check for `query.ExpressionAttributeNames` here to make sure it's what we expect.